.github/workflows/dapr.yml

```diff
@@ -30,9 +30,15 @@ on:
       - release-*
       - feature/*
 jobs:
-  build:
-    name: "Build and test on ${{ matrix.job_name }}"
-    runs-on: "${{ matrix.os }}"
+  unit-tests:
+    name: Unit test & lint
```

Comment:
Is it necessary to lint in each OS?

Reply:
At this moment, we have flakiness in some unit tests that show up mostly in darwin and windows. For that reason, I am keeping it on Linux first to begin with.

Reply:
Added it back. Now, unit tests run on darwin and windows.